### PR TITLE
Patch to add `box_corners` argument to `fc_draw()`

### LINKED ISSUE #2: option for boxes with corners?
It would be nice for some journals to have the option for boxes with corners

### PR DESCRIPTION
**Pull request by kenkomodo**
Summary of changes:
- Added `box_corners` argument to `fc_draw()` with options `"rounded"` or `"sharp"`
  - Default set to `"rounded"` to prevent change in behavior for current users who expect/want rounded corners from their existing scripts
- Updated `fc_draw()` documentation with new argument
- Fixed a typo in the vignette and added an example use of `box_corners` argument to vignette
- Updated `NEWS.md` with change
- Incremented version to 0.5.2

All checks passed locally with `devtools::check()`.

Resolves #2

### CODE REVIEW

File: vignettes/flowchart.Rmd
Comment (pasahe) [inline vignettes/flowchart.Rmd:1]: Thanks for fixing the typo! 

Regarding the example added with box_corners, I don't think it needs to be at the top of the customise output. I think it would fit better in example 4, where I change some function arguments. Also, I would like to add the box_corners argument explanation to the fc_draw() argument table within the 'Changing function arguments' section. Please fix this before merging.

File: R/fc_draw.R
Comment (pasahe) [inline R/fc_draw.R:1]: Thanks for the implementation. I like it a lot! The only thing is that I prefer the option to the argument to be "round" instead of "rounded" for simplicity. Please change that too.

File: DESCRIPTION
Comment (pasahe) [inline DESCRIPTION:1]: Please change the version back to development (0.5.1.9000) as I will not submit it to CRAN yet. The reason is that I think that there are not enough changes and also that the last submission was done recently.

File: NEWS.md
Comment (pasahe) [inline NEWS.md:1]: Please, change the header to development version.

**Review by pasahe — CHANGES_REQUESTED** [review]
Hi @kenkomodo,

Thanks for the pull request! I think having the ability to customise the box corners is a great addition to the package, and I love how you implemented it in the `fc_draw()` function. 

There are just a few small changes that need to be made for me to accept and merge the pull request:

- Replace the box_corner option "rounded" with "round" for simplicity.
- Move the location of the example use of the new argument to the Example 4 section.
- Change the version back to development 0.5.1.9000

Thanks for your great contribution!

File: NEWS.md
Comment (kenkomodo) [reply NEWS.md:1]: @pasahe Will do! 

I changed it to an explicit version just because `devtools::check()` on my computer didn't like the fact that "development version" wasn't at the top of the `NEWS.md` after I changed the ordering to match the current changelog.

File: R/fc_draw.R
Comment (kenkomodo) [reply R/fc_draw.R:1]: Makes sense to me!

File: DESCRIPTION
Comment (kenkomodo) [reply DESCRIPTION:1]: Done!

File: vignettes/flowchart.Rmd
Comment (kenkomodo) [reply vignettes/flowchart.Rmd:1]: Done!

**Comment by kenkomodo** [comment]
@pasahe I've pushed the requested edits.

**Review by pasahe — APPROVED** [review]
Awesome! Thanks for the quick response.

**Comment by pasahe** [comment]
The check seems to fail because it says that the docs files still have "rounded" instead of "round". If you use `document()` to update the docs and push it again, it should be fixed and ready to go.

**Comment by kenkomodo** [comment]
@pasahe  Oops! Knew I was forgetting something. I just fixed it.

**Comment by pasahe** [comment]
It's all good now. I have merged it into the main branch, so this version is now available for use. I really appreciate your time and contribution to this package!!

**Pull request merged**
